microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md

```diff
@@ -23,63 +23,55 @@ ms.date: 02/24/2023
+To make sure your website works for all your users, test your website on other browsers and devices. Not all of your users will be using the browser and device that you used when developing. We encourage you to check out solutions for emulation provided by other browsers, in addition to using the **Device Emulation** tool in Microsoft Edge DevTools.
 
 
 <!-- ------------------------------ -->
 #### Summary
 
-*  When you don't have a particular device, or want to do a spot check on something, the best option is to emulate the device right inside your browser.
+There are several different levels of testing that you can do to test your site under different conditions:
 
-*  Device emulators and simulators enable you to mimic your development site on a range of devices, from your workstation.
+* **Browser emulators** help you to quickly test how your website looks and behave at different screen sizes and resolutions, with touch event emulation, and by overriding the user-agent string the browser sends. You use browser emulators from the browser you're developing on. See [Browser emulators](#browser-emulators), below.
 
-*  Cloud-based emulators enable you to automate unit tests for your site across different platforms.
+* **Device emulators and simulators** enable you to run other devices from the device you're developing on and test your website on these other devices. See [Device emulators and simulators](#device-emulators-and-simulators), below.
+
+* **Cloud-based emulators** enable you to run automated tests for your site across different platforms. See [Cloud-based emulators and simulators](#cloud-based-emulators-and-simulators), below.
 
 
 <!-- ====================================================================== -->
 ## Browser emulators
 
-Browser emulators are great for testing the responsiveness of a site.  But a browser emulator doesn't emulate differences in API, CSS support, and certain behaviors that manifest only on a mobile browser on an actual device.  Test your site on browsers running on real devices, to be certain everything behaves as expected.
-
-
-<!-- ------------------------------ -->
-#### Firefox Responsive Design View
-
-Firefox has a [responsive design view](https://developer.mozilla.org/docs/Tools/Responsive_Design_View) that encourages you to stop thinking in terms of specific devices and instead explore how your design changes at common screen sizes, or on your own screen size by dragging the edges of the window.
+Browser emulators are great for testing the way your site looks and behaves at different screen sizes and resolutions, or when touch events are enabled, or when the user-agent string is set to another browser's user-agent string.
```

Comment:
```suggestion
Browser emulators are great for testing the way your site looks and behaves at different screen sizes and resolutions, or when touch events are enabled, or when the User-Agent string is set to another browser's User-Agent string.
```
hits 2 & 3 of 3: 
init cap per FTS @captainbrosset 
![image](https://github.com/user-attachments/assets/95395f27-5e87-407b-966a-fd3e6d5b1dbd)

Reply:
will push local commit